### DISCUSSION TITLE
31st open maintainers call

### DISCUSSION THREAD

**Discussion by mxinden** [libp2p/rust-libp2p#4713] [General]
Hi everyone,

The rust-libp2p maintainers will meet tomorrow (Tuesday) 09:00 UTC. See [libp2p public events calendar](https://calendar.google.com/calendar/embed?src=libp2p.io_0q9682i3te7eanhe9q7ae1c58g%40group.calendar.google.com).

https://meet.jit.si/rust-libp2p-open-maintainers-call

Everyone is welcome to join, listen along, propose topics and join the discussions.

Please post below in case you already want to add an agenda item or list it in [our notes document](https://hackmd.io/xoHKlFuER0OVj1DGDuFROw).

See you all tomorrow!

**Comment by thomaseizinger** [comment]
I'll join but I might run a bit late. Don't wait for me to start :)

**Comment by umgefahren** [comment]
Will there be an open maintainers call today?

Oh nevermind. I thought that they were scheduled for every two weeks.

**Reply by thomaseizinger** [reply]
They normally are but Max is at the IETF meeting in Prag and I couldn't attend either unfortunately so we cancelled it.
